client/index2.html

```diff
@@ -76,6 +76,7 @@
         </div>
```

Comment:
Index2 looks really cool and when the functionality is added it will look very professional

Reply:
Favicon isn't working for some reason.

Reply:
Yeah I'll get on that in a moment, I decided to go for a different image anyway